docs/FRSKY_R9.md

```diff
@@ -240,17 +242,19 @@ Wiring the receiver to the JR Bay on the back of your radio and flashing the ELR
 
 After flashing the ELRS bootloader, you can connect the serial port as described in the link above and use the ELRS UARTupload.py script to flash/update mLRS with latest mLRS firmware .elrs file. There are, of course, separate .elrs image files for each receiver:
 
+*Note: Python3 is a prerequisite to use the ELRS UARTUpload.py script.*
```

Comment:
why this in italics unlike all other comments? All italics?

Reply:
Italics removed.